py/server/deephaven/table.py

```diff
@@ -1304,6 +1304,43 @@ def agg_by(self, aggs: Union[Aggregation, Sequence[Aggregation]], by: Union[str,
         except Exception as e:
             raise DHError(e, "table agg_by operation failed.") from e
 
+    def partitioned_agg_by(self, aggs: Union[Aggregation, Sequence[Aggregation]],
```

Comment:
I think this PR should also include the new args to `agg_by`. `partitioned_agg_by` might make more sense when coupled with those.

Reply:
will include #2874 in this PR